java/openinference-semantic-conventions/src/main/java/io/openinference/semconv/trace/SpanAttributes.java

```diff
@@ -0,0 +1,116 @@
+package io.openinference.semconv.trace;
```

Comment:
let's also add io.openinference.semcove.resource or whatever it's called in Java.